### ISSUE TITLE
No dll file for windows servers

### ISSUE THREAD

**Issue by bleuscyther** [mysqludf/lib_mysqludf_json#7]
Need dll files for windows server 64/32 bit.

wasn't able to compile it succcessfully;

**Comment by charleyskira** [comment]
Agreed.  When following the directions on a windows server and trying to run the "CREATE FUNCTION" commands, I get the following error:

Error Code: 1126. Can't open shared library 'lib_mysqludf_json.so' (errno: 193 )

**Comment by uresh** [comment]
yes need dil file to work with windows. i caoundt find infor for windows.  please help ...
